### PR TITLE
Bugfixes for At This Page Mobile components

### PR DESCRIPTION
**Pull request by AndriiDovhopolov-okta**
## Description:
- Restyled 3rd party component
- Added showing actual location when page is opened in first time 
image

### Resolves:

* [OKTA-320804](https://oktainc.atlassian.net/browse/OKTA-320804)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/components/MobileOnThisPage.vue [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
   export default {
     name: 'MobileOnThisPage',
+    data: ()=>({ 
+     selectedOption: {}
+    }),
     computed: {
       options: function () {
         let optionsList = [];
         if( this.$page.headers ) {
+          console.log(this.$page.headers)
```
Comment (robertjd) [inline packages/@okta/vuepress-theme-prose/components/MobileOnThisPage.vue:25]: @AndriiDovhopolov-okta  this console log statement should be removed.

@bretterer @swiftone can we lint for these?

Comment (AndriiDovhopolov-okta) [reply packages/@okta/vuepress-theme-prose/components/MobileOnThisPage.vue:25]: Thanks, @robertjd missed that one.

**Comment by shuowu** [comment]
@AndriiDovhopolov-okta Can you add screenshots in PR's description to highlight the visual change?

**Comment by AndriiDovhopolov-okta** [comment]
image
@shuowu , here comes the screenshot.

**Comment by bretterer** [comment]
This PR has conflicts.  I can not merge @AndriiDovhopolov-okta 

```
packages/@okta/vuepress-theme-prose/assets/css/components/_onThisPage.scss
```

**Comment by bretterer** [comment]
This PR has conflicts. I can not merge @AndriiDovhopolov-okta

```
packages/@okta/vuepress-theme-prose/assets/css/components/_onThisPage.scss
```

**Pull request merged**